include/cutlass/epilogue/collective/intel_pvc_epilogue.hpp

```diff
@@ -0,0 +1,360 @@
+  can_implement(
+      ProblemShape const& problem_shape,
+      [[maybe_unused]] Arguments const& args) {
+    return true;
+  }
+
+  CUTLASS_HOST_DEVICE
+  CollectiveEpilogue(Params const& params_, TensorStorage const& shared_storage_)
+      : params(params_), fusion_callbacks(params_.thread, shared_storage_.thread) {}
+
+  CUTLASS_DEVICE
+  bool
+  is_producer_load_needed() const {
+    return fusion_callbacks.is_producer_load_needed();
+  }
+
+  template<
+    class ProblemShapeMNKL,
+    class TileShapeMNK,
+    class TileCoordMNKL,
+    class Accumulator,
+    class TiledMma,
+    class ResidueMNK
+  >
+  CUTLASS_DEVICE void
+  operator() (
+      ProblemShapeMNKL problem_shape_mnkl,
+      TileShapeMNK tile_shape_MNK,
+      TileCoordMNKL tile_coord_mnkl,
+      Accumulator accumulators, 
+      TiledMma tiled_mma,
+      ResidueMNK residue_mnk,
+      int thread_idx,
+      char* smem) {
+    
+    (void) tiled_mma;
+    (void) residue_mnk;
+    (void) smem;
+    using namespace cute;
+
+    static constexpr int DpasM = get<0>(shape(typename TiledMma::LayoutA_TV{})); // rows per dpas operation per sub_group for Matrix A
+    static constexpr int DpasN = get<1>(shape(typename TiledMma::LayoutB_TV{})); // cols per dpas operation per sub_group for Matrix B
+
+    static constexpr int FragsM = get<0>(EpilogueTile{}) / DpasM; // A frags per sub_group
+    static constexpr int FragsN = get<1>(EpilogueTile{}) / DpasN; // B frags per sub_group
+
+    static constexpr int FragmentSize = (DpasN * DpasM) / SubgroupSize;
+
+    // Indexing variables
+    auto [M, N, K, L] = problem_shape_mnkl;
+    auto [m_coord, n_coord, k_coord, l_coord] = tile_coord_mnkl;
+
+    bool is_C_load_needed = is_source_supported && fusion_callbacks.is_C_load_needed();
+
+    Tensor trC = make_tensor<typename TiledMma::ValTypeC>(Shape<Int<FragmentSize>>{});
+    Tensor trD = make_tensor<typename TiledMma::ValTypeD>(Shape<Int<FragmentSize>>{});
+    Tensor tOuti = params.xe_store_d.get_pvc_tensor(make_coord(m_coord, n_coord, 0),
+                                                  make_shape(Int<FragsM>{}, Int<FragsN>{}, L),
+                                                  make_stride(Int<DpasM>{}, Int<DpasN>{}));
+
+    Tensor rw_coord = tOuti(_,_,_,l_coord);
+    Tensor mD_crd = make_identity_tensor(make_shape(M,N));
+    Tensor cD = local_tile(mD_crd, take<0,2>(TileShapeMNK{}), make_coord(m_coord, n_coord));
+    // Get the fusion callbacks
+    constexpr bool RefSrc = true; // Register tensors reference R2S copy src layout
+    auto residue_mn = make_coord(M, N);
+    auto cst_args = cutlass::epilogue::fusion::detail::ConsumerStoreArgs{
+                      problem_shape_mnkl,
+                      TileShapeMNK{},
+                      tile_coord_mnkl,
+                      residue_mn,
+                      EpilogueTile{},
+                      params.xe_load_c,
+                      thread_idx,
+                      cD,
+                      cD,
```

Comment:
Should both of them to be `cD`?

Reply:
The second `cD` argument is supposed to represent the `per work item` share of the coordinates within a `sub-group`, but since the coordinates for PVC implementation are the same for all the `work items` in the `sub-group` we can pass the same argument twice.
Also, this argument doesn't get used for the `fusion::LinearCombination` operation but it is required for creating the generic arguments for the `ConsumerStore` interface, so once we have a use case for these coordinates we will be able to better adjust these arguments accordingly.